allennlp/models/semantic_parsing/wikitables/wikitables_erm_semantic_parser.py

```diff
@@ -229,14 +214,9 @@ def forward(self,  # type: ignore
             ``ProductionRule`` using a ``ProductionRuleField``.  We will embed all of these
             and use the embeddings to determine which action to take at each timestep in the
             decoder.
-        agenda : ``torch.LongTensor``
```

Comment:
I don't think you meant to remove this one.

Reply:
Also, up above, where I can't comment, you need to change the `WikiTablesWorld` type in the docstring (searching for that name in the whole repo would probably be a good idea).

Reply:
Done. Thanks!